lib/crossdoc/markdown_builder.rb

```diff
@@ -25,12 +25,18 @@ def build(content)
     def combine_text(children)
       children.map do |child|
         case child.type
-          when :em
-            "<em>#{child.children.first.value}</em>"
-          when :strong
-            "<strong>#{child.children.first.value}</strong>"
-          else
-            child.value
+        when :em
+          value = child.children.first.value.to_s
+          value.gsub!(/rsquo/, "'")
+          "<em>#{value}</em>"
+        when :strong
+          value = child.children.first.value.to_s
+          value.gsub!(/rsquo/, "'")
+          "<strong>#{child.children.first.value}</strong>"
+        else
+          value = child.value.to_s
+          value.gsub!(/rsquo/, "'")
+          value
```

Comment:
Rather than replacing them with straight quotes, we should replace them with the appropriate unicode character. Something like

```rb

    SMART_QUOTES = {
      lsquo: "\u2018", # ‘
      rsquo: "\u2019", # ’
      ldquo: "\u201C", # “
      rdquo: "\u201D", # ”
    }

    # combines the text of all child elements into one string
    def combine_text(children)
      children.map do |child|
        case child.type
          when :em
            "<em>#{combine_text(child.children)}</em>"
          when :strong
            "<strong>#{combine_text(child.children)}</strong>"
          when :smart_quote
            SMART_QUOTES[child.value]
          else
            child.value
        end
      end.join('')
    end
```